features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -2149,6 +2149,11 @@
     <!--This is the separator that use to separate multiple roles in the role claim value coming from IDP side-->
     <FederatedIDPRoleClaimValueAttributeSeparator>,</FederatedIDPRoleClaimValueAttributeSeparator>
 
+    <!-- Following applications can use federated user role based authorization -->
+    <FIdPRoleBasedAuthzApplications>
+        <AppName></AppName>
```

Comment:
does it work as expected with the empty config?

Reply:
yes, we check the list for SP name & having an empty list will also works here

Reply:
Since this is a sample value, can we have a commented XML tag with a sample app name